OpenRA.Mods.Common/Activities/ExternalCaptureActor.cs

```diff
@@ -76,6 +79,7 @@ public override Activity Tick(Actor self)
 							t.OnCapture(target.Actor, self, oldOwner, self.Owner);
 
 						capturable.EndCapture();
```

Comment:
This is done from inside `EndCapture`, so can be removed from here.

Reply:
done